addons/payment_demo/views/payment_demo_templates.xml

```diff
@@ -28,9 +28,11 @@
 
     <template id="express_checkout_form">
         <div name="o_express_checkout_container"
+             class="d-none"
```

Comment:
Why do you add this here, and not in Stripe?

Reply:
It is done differently in Stripe, the button is not loaded on the page if it's not needed (eg. when the amount is `0` or `undefined`). This is to prevent the button from "flash" when you enter a promo code that will give you cart with nothing left to pay, as we don't want the customer to use any of the express payment methods.

That said, now that you point it out, I wonder why the form is rendered (could be a t-if), and I also wonder what happens if you have a coupon for 1 product but two in the cart, and remove one of them (the amount should be 0 but there will be no refresh). I'll investigate.

Reply:
Glad it raised a potential issue.